File: Factorial.py
number = int(input("Enter the number to factorialize "))
total = 1
counter = number

if (number > 0) :
    while (counter > 1) :
        total = total * counter
        counter = counter - 1
    print(number,  "! = ",  total)
else :
    print("that is an invalid input")
          

File: Factorial.ts
var factorial = parseInt(prompt("Enter the number to factorialize"));
var counter = factorial;
var total = 1;

if (factorial > 0) {
	while (counter > 1) {
		total *= counter;
		counter--;
	}
	window.alert(factorial + "! = " + total);
} else {
	window.alert("That number is invalid");
}
